src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewModel.swift

```diff
@@ -62,12 +64,16 @@ class HealthCertificateOverviewViewModel {
 
 	func numberOfRows(in section: Int) -> Int {
 		switch Section(rawValue: section) {
-		case .createCertificate:
-			return 1
+		case .changeAdmissionScenarioStatusLabel:
+			return healthCertifiedPersons.isEmpty ? 0 : 1
```

Comment:
this logic will be replaced by task 11811
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11811